src/test/java/resourceTests/PersonResourceTest.java

```diff
@@ -97,4 +111,27 @@ public void testCreateNoteWithCorrectDate(){
 
         assertEquals(Response.Status.ACCEPTED.getStatusCode(), result.getStatus());
     }
+
+    @Test
+    public void testCreatePerson(){
+
+        resource.createPerson(firstPerson);
+
+        verify(mockPersonDao, times(1)).create(firstPerson);
+    }
+
+
+    @Test(expected = WebApplicationException.class)
+    public void testCreatePersonWithEmailInUse(){
+
+        when(mockPersonDao.findByEmail(firstPerson.getEmail())).thenReturn(mockList);
+
+        mockPersonDao.create(firstPerson);
+        resource.createPerson(secondPerson);
+
+        verify(mockPersonDao, times(1)).create(firstPerson);
```

Comment:
These verifications are never reached, this is dead code. Besides, the thrown `WebApplicationException` should have its status code checked.